### ISSUE TITLE
How to get started

### ISSUE THREAD

**Issue by dmansfield** [anjanpradhan/struts2-webflow#1]
Can you give a brief description in the README about how to get started using this plugin, including installing this plugin into struts2, and installing spring-web-flow into struts2?

**Comment by smchristensen** [comment]
Do you have a sample project I could take a look at.  I'm trying to move from webflow 1 with struts integration to webflow 2 and am hoping this will do the job.

**Comment by anjanpradhan** [comment]
@smchristensen yes, it will do the job of integrating Struts 2 and Spring Webflow 2. I am using it for few live projects. Having said that unfortunately, I don't have a sample project to demonstrate it. I might try to create one.

**Comment by mohitraj1** [comment]
did anyone had luck getting it to work in their project. If so, please share the configs needded

**Comment by anjanpradhan** [comment]
@mohitraj1 yes there are quite a few projects (I know) use this to integrate Struts 2 and Spring Webflow 2. I am not sure if  I can share those configuration. Rather, I can try to build an example application and share.
